src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -138,9 +138,14 @@ endif()
 # MFEM
 #------------------------------------------------------------------------------
 if (TARGET mfem)
-    # Case: included axom in project that also creates an mfem target, no need to recreate mfem
+    # Case: Axom included in project that also creates an mfem target, no need to recreate mfem
+    # Note - white238: I can't seem to get this to pass install testing due to mfem being included
+    # in multiple export sets
     message(STATUS "MFEM support is ON, using existing mfem target")
-    blt_list_append(TO TPL_DEPS ELEMENTS mfem)
+    # Add it to this export set but don't prefix it with axom::
```

Comment:
I'm sure you played around with this, but would it make sense for axom to export this with the `axom::` prefix, but for the "host code" (of which axom and mfem are submodules) to export it as `mfem` ?

Specifically, my suggestion here is for axom to export the target as `axom::mfem` whether it was imported or already a target.

Reply:
In this case, `mfem` was a real target not an imported target and CMake had checks that it did not exist in multiple exported configs.